Author: Michael R. Crusoe <crusoe@debian.org>
Date:   Wed Mar 27 10:16:56 2024 +0100
Forwarded:

    Marks some doctstrings as raw, to silence a SyntaxWarning about invalid escape sequences.

    Before
    ``` sh
    $ python -c 'import rdflib.extras.external_graph_libs'
    rdflib/extras/external_graph_libs.py:79: SyntaxWarning: invalid escape sequence '\*'
      """Converts the given graph into a networkx.MultiDiGraph.
    rdflib/extras/external_graph_libs.py:136: SyntaxWarning: invalid escape sequence '\*'
      """Converts the given graph into a networkx.DiGraph.
    rdflib/extras/external_graph_libs.py:199: SyntaxWarning: invalid escape sequence '\*'
      """Converts the given graph into a networkx.Graph.
    $
    ```

    After:
    ``` sh
    $ python -c 'import rdflib.extras.external_graph_libs'
    $
    ```

--- rdflib.orig/rdflib/extras/external_graph_libs.py
+++ rdflib/rdflib/extras/external_graph_libs.py
@@ -77,7 +77,7 @@
 def rdflib_to_networkx_multidigraph(
     graph: Graph, edge_attrs=lambda s, p, o: {"key": p}, **kwds
 ):
-    """Converts the given graph into a networkx.MultiDiGraph.
+    r"""Converts the given graph into a networkx.MultiDiGraph.
 
     The subjects and objects are the later nodes of the MultiDiGraph.
     The predicates are used as edge keys (to identify multi-edges).
@@ -134,7 +134,7 @@
     edge_attrs=lambda s, p, o: {"triples": [(s, p, o)]},
     **kwds,
 ):
-    """Converts the given graph into a networkx.DiGraph.
+    r"""Converts the given graph into a networkx.DiGraph.
 
     As an rdflib.Graph() can contain multiple edges between nodes, by default
     adds the a 'triples' attribute to the single DiGraph edge with a list of
@@ -197,7 +197,7 @@
     edge_attrs=lambda s, p, o: {"triples": [(s, p, o)]},
     **kwds,
 ):
-    """Converts the given graph into a networkx.Graph.
+    r"""Converts the given graph into a networkx.Graph.
 
     As an rdflib.Graph() can contain multiple directed edges between nodes, by
     default adds the a 'triples' attribute to the single DiGraph edge with a
